src/main/resources/assets/thaumcraft/lang/en_us.lang

```diff
@@ -0,0 +1,328 @@
+#
+research.SEALSTOCK.stage.2=This seal is similar to the store seal, except it keeps a certain number of items stocked inside its container at all times. The seal only ensures that a minimum number of items are kept in stock - it will not prevent overstocking should it occur by other means.<BR>This seal requires provider seals to know where to get the items.
+#
+research.GOLEMFLYER.stage.2=By essentially chopping off a golem's legs and gluing an Arcane Levitator in their place, I have managed to get a golem to fly! This grants it good, if slightly erratic mobility, though it does move more slowly.<BR>§9This research unlocks additional golem parts§0.
+research.f_FLY.text=Closely examine a flying creature
+#
+research.GOLEMBREAKER.title=Golem Manipulation: Block Breaking
+research.GOLEMBREAKER.stage.2=By replacing a golem's arms with a set of powerful, diamond-tipped pneumatic grinders it will be able to break through all but the hardest materials.<BR>§9This research unlocks additional golem parts.§0
+#
+research.GOLEMCOMBATADV.title=Improved Golem Combat
+research.GOLEMCOMBATADV.stage.2=I can now craft specialized golem weaponry and armor to make them much more effective in combat.<BR>§9This research unlocks additional golem parts.§0
+#
+
+##########################################
+# ELDRITCH
+##########################################
+#
+research.BASEELDRITCH.stage.1=The void seed I have discovered begs further study. I should study it further and perhaps discover a practical use for it.<BR>Currently it seems morphically unstable - its physical properties are constantly shifting and changing. Maybe binding it with something else will make it easier to manipulate. Metal perhaps?
+research.BASEELDRITCH.stage.2=My hand shakes as I write this. The strange substance I have investigated has opened strange new vistas in my mind.<BR>Before me I have formulas for creating a mysterious new metal, yet I do not remember writing them, nor does the handwriting look quite like mine.<BR>The metal they describe is a mass of contradictions. It is hard and unyielding, yet at the same time soft and easy to work. It will feel warm to the touch, almost alive, and like living things it will slowly heal itself if damaged.<BR>Crafting mere physical tools from this seems almost... wrong.
+#
+research.VOIDSIPHON.stage.2=I hate to admit it, but the Crimson Cult might be on to something. I have managed to design a Void Siphon that slowly leeches energy from nearby rifts. This leeched energy will slowly form into Void Seeds. The larger the rift is, the quicker this process occurs.<BR>Unfortunately this is not without consequence for the rift itself. The rift will become increasingly unstable and will slowly shrink as its energy is leeched away. I must take care to not overuse them.<BR>A redstone signal will disable the Void Siphon.
```

Comment:
maybe use "leached" instead of "leeched"? Not sure which is more appropriate here.

Reply:
Leech is like the animal, leached is a verb, so yes that needs changing.